modules/culturefeed_search_api/src/DrupalCulturefeedSearchClient.php

```diff
@@ -245,4 +250,20 @@ public function getRegions() {
     return $this->staticCache[$cid];
   }
 
+  /**
+   * Alter a Culturefeed search query before it is executed.
+   *
+   * @param \CultuurNet\SearchV3\SearchQueryInterface $searchQuery
+   *   The search query to alter.
+   * @param string $type
+   *   The type of query that is executed. Can be one of the following:
+   *   - events
+   *   - event
+   *   - places
+   *   - offers.
+   */
+  protected function alterQuery(SearchQueryInterface $searchQuery, $type = 'events') {
+    \Drupal::moduleHandler()->alter('culturefeed_search_api_query', $searchQuery, $type);
```

Comment:
No use of dependency injection here. Please adjust.

Reply:
Merged, going to update the dependency injection on our branch